Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM php:7.0-apache
+
+RUN apt-get update \
```

Comment:
This is doing exactly three things:

- Ensuring we have the extensions necessary to run composer.
- Ensuring the web server is configured correctly (enables mod_rewrite and changes the document root).
- Installing composer.

I'm not sure we really need to simplify it further…